README.md

```diff
@@ -1,2 +1,8 @@
-sulu-docs
-=========
+Sulu Documentation
+==================
+
+The [Sulu](http://sulu.io) Documentation can be found on (http://docs.sulu.io).
```

Comment:
ha, i know why i did this. because it's md not restructured text. it's just for github.